website/docs/core-concepts/validate/json-schema.mdx

```diff
@@ -42,7 +42,17 @@ schemas:
 
 In the component [manifest](https://github.com/cloudposse/atmos/blob/master/examples/quick-start-advanced/stacks/catalog/vpc/defaults.yaml), add
 the `settings.validation` section:
-
+### Specify Remote Schemas
+You can specify remote schemas by setting the `manifest` field to a remote URL in your `atmos.yaml` configuration file.
+<File title="atmos.yaml">
+```yaml
+# Validation schemas (for validating atmos stacks and components)
+schemas:
+  atmos:
+    # You can specify a remote schema URL as well
+    manifest: "https://atmos.tools/schemas/atmos/atmos-manifest/1.0/atmos-manifest.json"
+```
+</File>
```

Comment:
```suggestion
### Use Remote Schemas

You can specify remote schemas by setting the `manifest` field to a remote URL in your `atmos.yaml` configuration file.

<File title="atmos.yaml">
```yaml
# Validation schemas (for validating atmos stacks and components)
schemas:
  atmos:
    # You can specify a remote schema URL as well
    manifest: "https://atmos.tools/schemas/atmos/atmos-manifest/1.0/atmos-manifest.json"
```
</File>
```